docs/containers.conf.5.md

```diff
@@ -47,341 +47,403 @@ The containers table contains settings pertaining to the OCI runtime that can
 configure and manage the OCI runtime.
 
 **devices**=[]
-  List of devices.
-  Specified as 'device-on-host:device-on-container:permissions',
-  for example: "/dev/sdc:/dev/xvdc:rwm".
+
+List of devices.
```

Comment:
What does the man page look like after you do this?

Reply:
All the podman manpages are formatted like this, with the empty newline. Here's what they look like as a manpage:

![Screenshot from 2020-08-03 14-20-21](https://user-images.githubusercontent.com/32910034/89214259-7bf8cb00-d594-11ea-855c-79d46382d8be.png)